roles/contrail_common/tasks/determine_contrail_masters.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Set contrail_masters var
+  set_fact:
+    contrail_masters: ""
```

Comment:
This var should be defined in `roles/contrail_common/defaults/mail.yml`

Reply:
will resolve this on my next commit